js/scripts.js

```diff
@@ -0,0 +1,107 @@
+var phones,
+    btnRing = document.getElementsByClassName('btn-ring'),
+    btnText = document.getElementsByClassName('btn-text'),
+    btnPhoto = document.getElementsByClassName('btn-photo'),
+    paragraphs = document.getElementsByTagName('p');
+
+// ---------------------------------------------------------------------------------------------------
+
+function Phone(brand, model, price, color, screenSize, resolH, resolV) {
```

Comment:
w najprostszej wersji kod wyglądał by następująco:
```
function Phone(phoneSpec) {
  this.brand = phoneSpec.brand;
  this.model = phoneSpec.model;
  ...
}
```

a wywołanie wyglądało by następująco:
```
var iphone = new Phone({
  brand: "Apple",
  model: "iPhone 6s",
  ...
})
```

Bardziej zaawansowana wersja (z użyciem object destructing) wyglądała by następująco:
```
function Phone({brand, model, ...}) {
  this.brand = brand;
  this.model = model;
  ...
}
```
a wywołanie pozostaje bez zmian względem "prostszej" wersji. Zaletą tego rozwiązania jest to, że nie ma znaczenia kolejność argumentów, ponieważ jest tylko jeden argument - obiekt z właściwościami, które są nazwane. (http://2ality.com/2011/11/keyword-parameters.html)